test/e2e_tests/e2e_test.py

```diff
@@ -0,0 +1,81 @@
+from multiprocessing.sharedctypes import Value
+import pytest
+import os
+from threading import Thread
+from todo_app import app
+from dotenv import load_dotenv, find_dotenv
+from todo_app.data.trello_items import TrelloItems
+from selenium import webdriver
+from selenium.webdriver.firefox.options import Options
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import NoSuchElementException
+
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    file_path = find_dotenv("../.env")
+    load_dotenv(file_path, override=True)
+
+    trello_items = TrelloItems()
+    board_id = trello_items.create_board("test1234")
+    os.environ['TRELLO_BOARD_ID'] = board_id
+    
+    application = app.create_app()
+    thread = Thread(target=lambda: application.run(use_reloader=False))
+    thread.daemon = True
+    thread.start()
+    yield application
+
+    thread.join(1)
+    trello_items.delete_board(board_id)
+
+@pytest.fixture(scope="module")
+def driver():
+    options = Options()
+    options.binary_location = "C:/Program Files/Mozilla Firefox/firefox.exe"
```

Comment:
This is slightly sad because it relies on the machine running it being Windows, and all developers using the same folder structure. If you make sure that the `Mozilla Firefox` folder is in your PATH environment variable, you should find you can remove this line entirely